src/styles/card.scss

```diff
@@ -0,0 +1,75 @@
+.card {
+  width: 198px; // including border
+  height: 406px;
+  border: 1px solid $card-border-color;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+
+  &__image {
+    padding: 32px 19px 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 150%;
+    color: $card-title-color;
+    padding-bottom: 4px;
+    padding-left: 16px;
+    padding-right: 16px;
+    max-width: 250px;
+    word-wrap: break-word;
+  }
+
+  &__code {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 140%;
+    color: $card-code-color;
+    padding-bottom: 16px;
+    padding-left: 16px;
+    padding-right: 16px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    font-size: 12px;
+    line-height: 112%;
+    padding-bottom: 24px;
+    padding-left: 16px;
+    padding-right: 16px;
+    color: $card-price-color;
+
+    &-number {
+      font-weight: 700;
+      font-size: 16px;
+    }
+  }
+
+  &__link {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+    width: 166px;
+    height: 40px;
+    margin: 0 auto;
+    padding: 12px 0;
+    margin-bottom: 16px;
+    border-radius: 5px;
+    background-color: $card-link-bg-color;
+    color: $card-link-hover-color;
+    line-height: 114%;
+    text-decoration: none;
+    font-size: 14px;
+    font-weight: 700;
+
+    &:hover {
+      background-color: $background-color;
```

Comment:
The `background-color` property in the `&:hover` state should be set to `$card-link-hover-bg-color` instead of `$background-color` to follow the variable naming convention and to avoid confusion with other background color usages in the styles.